src/tests/ftest/container/auto_oc_selection.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from apricot import TestWithServers
+from test_utils_container import TestContainer
+from avocado.core.exceptions import TestFail
+
+
+class AutoOCSelectionTest(TestWithServers):
+    """Automatic object class selection test.
+
+    Test Class Description:
+    Create a container with different RF and OC combinations. We expect some
+    combinations to fail due to the restriction set by DAOS. (This is more like
+    auto OC "restriction" rather than auto OC "selection".)
+
+    :avocado: recursive
+    """
+    def test_oc_restrictions(self):
```

Comment:
Question - Is there a reason for mixing "restriction" and "selection"? Could we simplify and use just one term?
- Filename `auto_oc_selection.py`
- Class name `AutoOCSelectionTest`
- Test method name `test_oc_restrictions`
- Test tag `oc_restrictions`

Reply:
Done